### PR TITLE
Update backgrid module to be named

### PR DESCRIPTION
**Pull request by kluo**
This fixes the broken loading... UI upon saving the studio editor
@stvstnfrd

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -81,3 +81,5 @@ update-npm-requirements:
 	npm update --silent
 	cp ./node_modules/backgrid/lib/backgrid*.js $(STATIC_JS)/lib/backgrid/
 	cp ./node_modules/backgrid/lib/backgrid*.css $(STATIC_CSS)/lib/backgrid/
+	sed -i 's/\["underscore"/"backgrid", &/' $(STATIC_JS)/lib/backgrid/backgrid.js
```
Comment (stvstnfrd) [inline Makefile]: Is this just doing both the non-minified as well as the minified copy for thoroughness?

Comment (kluo) [reply Makefile]: yep, functionally it only uses `backgrid.min.js` for the ultimate minified js file

**Comment by stvstnfrd** [comment]
@kluo Do we need to bump the version number of the XBlock?

Also, what changes do we need to make to platform?

**Comment by kluo** [comment]
Hm, well technically this fix is part of the same group of changes for the gingko update, which you already bumped the version for, so I think it should be fine as is

**Pull request merged**